### ISSUE TITLE
TTv5 Q&A: Checkboxes do not have appropriate markup.

### ISSUE THREAD

**Issue by HD508Github** [Section508Coordinators/DHSA-TT-200-A#170]
Location: Search page
URL: https://dev.section508testing.net/mod/forum/search.php?search=&id=89
Issue: ANDI Output does not identify an accessible name, associated label or title attribute for the checkboxes.  Also, 'ALT' attributes are inappropriately assigned to the checkboxes. 

![Checkboxes (Search page)](https://user-images.githubusercontent.com/34749031/59941706-e6e85080-942b-11e9-8b5d-aae3ff2fb11f.jpg)

**Comment by drewnielson** [comment]
Hey @katbecker, this looks like an issue we'll need to send up the chain for a potential fix from Moonami. Not sure how we've gone about that in the past, so sending your way for guidance. We might want to discuss with Cindy before we send it along in any case. Confirmed that code is using an alt attribute on checkboxes, which is not valid code.

**Comment by katbecker** [comment]
Sadly, Moodle lacks a strong a11y group. I'll see what can be done. We should be able to manipulate the plug in php with a hook to add an alt (I take it you're putting in a forum with advanced search).

**Comment by drewnielson** [comment]
Right, adding in a forum with Advanced search. Let's hold off until after the upgrade to version 3.5, then re-test to see if it is still an issue.
